integration_test/gce/gce_testing.go

```diff
@@ -377,27 +377,28 @@ func lookupMetric(ctx context.Context, logger *log.Logger, vm *VM, metric string
 
 // hasNonEmptySeries examines the given iterator, returning true if the
 // lookup succeeded and returned a nonempty time series, and false otherwise.
-// Also returns an error if the lookup failed.
-// A return value of (false, nil) indicates that the lookup succeeded but
+// Also returns an error if the lookup failed, as well as the time series
+// itself if the lookup succeeded.
+// A return value of (false, nil, nil) indicates that the lookup succeeded but
 // returned no data.
-func hasNonEmptySeries(logger *log.Logger, it *monitoring.TimeSeriesIterator) (bool, error) {
+func hasNonEmptySeries(logger *log.Logger, it *monitoring.TimeSeriesIterator) (bool, error, *monitoringpb.TimeSeries) {
```

Comment:
Don't you now have the invariant that the third return value is `nil` iff the first return value is `false`? How about renaming this to `getNonEmptySeries` (or just `nonEmptySeries`) and having it return just `(*monitoringpb.TimeSeries, error)`?

[Optional nit] You use "nonempty" as one word below, which means this should be `hasNonemptySeries`, right?